src/account.ts

```diff
@@ -1,6 +1,11 @@
 const assert = require('assert')
+const {
+  privateKeyVerify,
+  publicKeyCreate,
+  publicKeyVerify,
+  publicKeyConvert,
+} = require('ethereum-cryptography/shims/hdkey-secp256k1v3')
```

Comment:
Why do we import these shims? Those shouldn't be considered public and will be removed once `hdkey` gets updated.